### PR TITLE
Loadflow provider default value can now be defined in application.properties

### PR DESCRIPTION
**Pull request by archendel**
Signed-off-by: LE SAULNIER Kevin <kevin.lesaulnier@rte-france.com>

### CODE REVIEW

File: src/main/resources/application-local.yaml [hunk 1]
```diff
@@ -11,3 +11,6 @@ network-store-server:
 actions-server:
   base-uri: http://localhost:5022
 
+# default values for loadflow providers "Hades2" or "OpenSecurityAnalysis"
+loadflow:
+  default-provider: OpenSecurityAnalysis
```
Comment (FranckLecuyer) [inline src/main/resources/application-local.yaml:16]: Missing newline

File: src/main/java/org/gridsuite/securityanalysis/server/util/SecurityAnalysisRunnerSupplier.java [hunk 2]
```diff
@@ -6,21 +6,25 @@
...
 /**
  * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
  */
-public final class SecurityAnalysisUtil {
+@Configuration
+public class SecurityAnalysisRunnerSupplier {
 
-    private SecurityAnalysisUtil() {
-    }
+    @Value("${loadflow.default-provider:OpenSecurityAnalysis}")
```
Comment (FranckLecuyer) [inline src/main/java/org/gridsuite/securityanalysis/server/util/SecurityAnalysisRunnerSupplier.java:20]: security-analysis.default-provider

File: src/main/resources/application-local.yaml [hunk 3]
```diff
@@ -11,3 +11,6 @@ network-store-server:
 actions-server:
   base-uri: http://localhost:5022
 
+# default values for loadflow providers "Hades2" or "OpenSecurityAnalysis"
+loadflow:
```
Comment (FranckLecuyer) [inline src/main/resources/application-local.yaml:15]: security-analysis:

File: src/test/java/org/gridsuite/securityanalysis/server/util/SecurityAnalysisUtilTest.java [hunk 4]
```diff
@@ -8,20 +8,33 @@
...
 import static org.junit.Assert.*;
 
 /**
  * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
  */
+@RunWith(SpringRunner.class)
+@SpringBootTest
 public class SecurityAnalysisUtilTest {
 
+    @Value("${loadflow.default-provider:OpenSecurityAnalysis}")
```
Comment (FranckLecuyer) [inline src/test/java/org/gridsuite/securityanalysis/server/util/SecurityAnalysisUtilTest.java:26]: security-analysis.default-provider

File: src/test/java/org/gridsuite/securityanalysis/server/util/SecurityAnalysisUtilTest.java [hunk 5]
```diff
@@ -8,20 +8,33 @@
...
+    String defaultLoadflowProvider;
```
Comment (FranckLecuyer) [inline src/test/java/org/gridsuite/securityanalysis/server/util/SecurityAnalysisUtilTest.java:27]: defaultProvider

**Review by FranckLecuyer — CHANGES_REQUESTED** [review]

File: src/main/resources/application-local.yaml (hunk 1 above)
Comment (archendel) [reply src/main/resources/application-local.yaml:16]: property moved to application.yaml, with new line added

**Review by FranckLecuyer — APPROVED** [review]

**Pull request merged**